src/tools/run-make-support/src/run.rs

```diff
@@ -92,3 +106,11 @@ pub fn cmd<S: AsRef<OsStr>>(program: S) -> Command {
     command.env("LC_ALL", "C"); // force english locale
     command
 }
+
+fn split_maybe_args(s: &str) -> Vec<OsString> {
+    s.split(' ')
+        .filter_map(|s| {
+            if s.chars().all(|c| c.is_whitespace()) { None } else { Some(OsString::from(s)) }
+        })
+        .collect()
+}
```

Comment:
Suggestion: can you leave me a FIXME to do proper env var / shell arg splitting?

Reply:
Of course, just a `FIXME(#132599)` pointing to #132599 😁